### PR TITLE
Error message layout

### PR DESCRIPTION
**Pull request by FanQueen**
Add ErrorMessage styled component, fix layout by height

### CODE REVIEW

File: src/components/SoundBox/SoundBox.js [hunk 1]
```diff
@@ -27,7 +28,7 @@ class SoundBox extends Component {
 
   render() {
     const { clock, soundBank } = this;
-    if (clock) {
+    if (false) {
```
Comment (barsukov) [inline src/components/SoundBox/SoundBox.js]: ?

**Review by barsukov — CHANGES_REQUESTED** [review]
Would be nice if you will add a test for this behavior.

File: src/components/SoundBox/SoundBox.test.js [hunk 2]
```diff
@@ -14,4 +15,9 @@ describe('SoundBox', () => {
     expect(renderedSoundBox.find(SandButton)).toHaveLength(4);
     expect(renderedSoundBox.find(VioletButton)).toHaveLength(4);
   });
+  it('renders error message if browser does not support Web Audio', () => {
```
Comment (barsukov) [inline src/components/SoundBox/SoundBox.test.js]: Please add another for webkit also

File: src/components/SoundBox/SoundBox.test.js [hunk 3]
```diff
@@ -15,6 +15,11 @@ describe('SoundBox', () => {
     expect(renderedSoundBox.find(SandButton)).toHaveLength(4);
     expect(renderedSoundBox.find(VioletButton)).toHaveLength(4);
   });
+  it('renders SoundBox if browser uses webkitAudioContext', () => {
+    window.AudioContext = window.AudioContext || window.webkitAudioContext;
```
Comment (barsukov) [inline src/components/SoundBox/SoundBox.test.js]: Please tell me the purpose of this code? I delete this line and the test will be green

Comment (FanQueen) [reply src/components/SoundBox/SoundBox.test.js]: I fixed this in next commit

**Review by barsukov — APPROVED** [review]
Good job 👍

**Comment by barsukov** [comment]
Next time I think we can refactor it to using the React Error Boundaries

**Pull request merged**